src/scriptdev2/scripts/northrend/borean_tundra.cpp

```diff
@@ -1195,6 +1196,126 @@ CreatureAI* GetAI_npc_seaforium_depth_charge(Creature* pCreature)
+    SAY_1_ELDER_MOOTOO      = -1001236,
+    SAY_2_ELDER_MOOTOO      = -1001238,
+
+    NPC_ELDER_MOOTOO        = 25503,
+
+    QUEST_ESCAPING_THE_MIST = 11664
+};
+
+struct npc_mootoo_the_youngerAI : public npc_escortAI
+{
+    npc_mootoo_the_youngerAI(Creature* pCreature) : npc_escortAI(pCreature) { Reset(); }
+
+    void Reset() override
+    {
+        if (!HasEscortState(STATE_ESCORT_ESCORTING))
+            m_creature->SetStandState(UNIT_STAND_STATE_SIT);
+    }
+
+    void WaypointReached(uint32 uiPointId) override
+    {
+        switch (uiPointId)
+        {
+            case 1:
+                m_creature->SetStandState(UNIT_STAND_STATE_STAND);
+                if (Player* pPlayer = GetPlayerForEscort())
+                    DoScriptText(SAY_MOOTOO_Y_START, m_creature, pPlayer);
+                break;
+            case 4:
+                DoScriptText(SAY_1_MOOTOO_Y, m_creature);
+                break;
+            case 9:
+                if (Player* pPlayer = GetPlayerForEscort())
+                    DoScriptText(SAY_2_MOOTOO_Y, m_creature, pPlayer);
+                break;
+            case 10:
+                m_creature->HandleEmote(EMOTE_ONESHOT_POINT);
+                break;
+            case 12:
+                DoScriptText(SAY_3_MOOTOO_Y, m_creature);
+                break;
+            case 14:
+                DoScriptText(SAY_4_MOOTOO_Y, m_creature);
+                break;
+            case 15:
+                DoScriptText(SAY_5_MOOTOO_Y, m_creature);
+                break;
+            case 16:
+                DoScriptText(SAY_6_MOOTOO_Y, m_creature);
+                break;
+            case 18:
+                DoScriptText(SAY_4_MOOTOO_Y, m_creature);
+                break;
+            case 19:
+                DoScriptText(SAY_7_MOOTOO_Y, m_creature);
+                break;
+            case 20:
+                DoScriptText(SAY_8_MOOTOO_Y, m_creature);
+                break;
+            case 22:
+                DoScriptText(SAY_CREDIT_MOOTOO_Y, m_creature);
+                SetRun();
+                break;
+            case 23:
+                if (Player* pPlayer = GetPlayerForEscort())
+                    pPlayer->GroupEventHappens(QUEST_ESCAPING_THE_MIST, m_creature);
+                if (Creature* pFather = GetClosestCreatureWithEntry(m_creature, NPC_ELDER_MOOTOO, 30.0f))
+                    DoScriptText(SAY_1_ELDER_MOOTOO, pFather);
+                break;
+            case 24:
+                DoScriptText(SAY_9_MOOTOO_Y, m_creature);
+                if (Player* pPlayer = GetPlayerForEscort())
+                if (Creature* pFather = GetClosestCreatureWithEntry(m_creature, NPC_ELDER_MOOTOO, 30.0f))
+                    DoScriptText(SAY_2_ELDER_MOOTOO, pFather, pPlayer);
+                break;
+            case 25:
+                if (Player* pPlayer = GetPlayerForEscort())
+                if (Creature* pFather = GetClosestCreatureWithEntry(m_creature, NPC_ELDER_MOOTOO, 30.0f))
```

Comment:
Add some extra spaces here, or surround by { .... }. It makes the code look better and cleaner.
Same for above.

Everything else looks good.